.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: php
+
+before_script:
+  - wget https://getcomposer.org/composer.phar
+  - chmod +x composer.phar
+  - ./composer.phar update nothing
+  - ./composer.phar install
+
+
+script:
```

Comment:
You will probably need a token in codacy. ping @nanojover

Reply:
There is no dependencies to download so far.  I think we don't need the token yet. @nanojover ?

Reply:
checked